pyproject.toml

```diff
@@ -65,8 +65,8 @@ dependencies = [
     "rasterio>=1.3.3,!=1.4.0,!=1.4.1,!=1.4.2",
     # rtree 1.0.1+ required for Python 3.11 wheels
     "rtree>=1.0.1",
-    # segmentation-models-pytorch 0.3.3+ required for timm 0.8+ support
-    "segmentation-models-pytorch>=0.3.3",
+    # segmentation-models-pytorch 0.5+ required for new UnetDecoder API
+    "segmentation-models-pytorch>=0.5",
```

Comment:
It's prohibitively difficult to support both APIs, let's only support SMP 0.5+

Reply:
that's what I'm talking about